spec/helpers/transactions_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the TransactionsHelper. For example:
+#
+# describe TransactionsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe TransactionsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
- [ ] Kindly add tests for your file or delete the test file, it's not good practice to have test files without files.